test/unit/controller/placeholder.js

```diff
@@ -66,147 +66,146 @@ module.exports.tests.should_execute = (test, common) => {
 };
 
 module.exports.tests.success = (test, common) => {
-  test('response from service should be converted', (t) => {
-    const logger = mock_logger();
-
-    const placeholder_service = (req, callback) => {
-      t.deepEqual(req, { param1: 'param1 value' });
-
-      const response = [
+  const response = [
```

Comment:
this `$response` object was simply moved up one scope without modification so it could be referenced below.